pkg/restore/pod_action_test.go

```diff
@@ -169,17 +169,18 @@ func TestPodActionExecute(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			action := NewPodAction(arktest.NewLogger())
 
-			res, warning, err := action.Execute(test.obj, nil)
-
-			assert.Nil(t, warning)
+			res, err := action.Execute(&RestoreItemActionExecuteInput{test.obj, test.obj, nil})
 
 			if test.expectedErr {
 				assert.NotNil(t, err, "expected an error")
 			} else {
 				assert.Nil(t, err, "expected no error, got %v", err)
 			}
 
-			assert.Equal(t, test.expectedRes, res)
+			if res != nil {
```

Comment:
I think it's stronger/safer to assert/require that res is non-nil when test.expectedErr is false?